ops/terraform/dev/us-central1/01_library_oauth_client/terragrunt.hcl

```diff
@@ -0,0 +1,31 @@
+# Terraform module call to configure datastore for LibraryViewDoc Entity kind
+terraform {
+  source = "${get_parent_terragrunt_dir()}//modules//library_oauth_client"
+}
+
+include {
+  path = find_in_parent_folders()
+}
+
+# Set a dependency for the GCP APIs being enabled, which will prevent this module from running too early
+dependency "library_service_api" {
+  config_path  = "../00_library_service_api"
+  skip_outputs = true
+}
+
+
+locals {
+  environment     = "dev"
+  resource_prefix = "nytimes-library"
+  project_id      = read_terragrunt_config(find_in_parent_folders("account.hcl"))
+}
+
+inputs = {
+  project_id = local.project_id.locals.project_id
+  # When calling this module ensure you are the owner of the provided google group!
+  support_email                  = "infrastructure.admin@adhocteam.us"
```

Comment:
it should be generally the same one, as if you're configuring the oauth client on the GUI

For prod we'd import the already created resources as much as possible to avoid needing to recreate the entire thing

Reply:
interesting --- what is the identifier that helps match this up with the existing prod one?